floodlight/io/dfl.py

```diff
@@ -375,25 +375,33 @@ def read_event_data_xml(
     tree = etree.parse(str(filepath_events))
     root = tree.getroot()
 
-    # find start and end of half
-    kickoff_whistles = {}
-    final_whistles = {}
-    for whistle in root.findall("Event/KickoffWhistle"):
-        if whistle.get("GameSection") is not None:
-            kickoff_whistles[whistle.get("GameSection")] = iso8601.parse_date(
-                whistle.getparent().get("EventTime")
+    # find start of halves
+    start_times = {}
+    start_events = root.findall("Event/KickoffWhistle")
```

Comment:
maybe a comment would be helpful that this is a looking for two different encodings of whistles as the data format changed over time